lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -191,6 +191,14 @@ div.course-wrapper {
         margin: 0 auto;
       }
     }
+    .disable-proctored-exam-btn {
+        i {
+          color: rgb(206, 216, 220);
```

Comment:
please use one of the pre-defined color variables in _variables.scss (lms) for this istead of directly defining RGB.